docs/studio/rbac.mdx

```diff
@@ -6,16 +6,14 @@ icon: "user-tie"
 
 Role-Based Access Control (RBAC) is a security model that plays a crucial role in managing access to resources within an organization. RBAC simplifies access control by assigning permissions to roles rather than individuals. Users are then associated with specific roles, and their access rights are determined by the permissions linked to those roles. RBAC is widely used in various industries to establish a robust and structured access control system.
 
-The roles provided in Cosmo are as follows
```

Comment:
We should create a separate page "Members" or similar, where we explain the basics but also that you can invite users to groups.